webpack.dev.config.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const Merge = require('webpack-merge');
+const path = require('path');
+const webpack = require('webpack');
+
+const commonConfig = require('./webpack.common.config.js');
+
+module.exports = Merge.smart(commonConfig, {
+    output: {
+        filename: '[name].js'
+    },
+    devtool: 'source-map',
+    plugins: [
+        new webpack.LoaderOptionsPlugin({
+            debug: true
+        })
+    ],
+    module: {
+        rules: [
+            {
+                test: /(.scss|.css)$/,
+                include: [
+                    /studio-frontend/,
+                    /paragon/,
+                    /font-awesome/
+                ],
+                use: [{
```

Comment:
I'm very unsure about this `use` clause; it's basically the same as prod, but extracted from the `extractSass.extract({` wrapper.

Reply:
Out of curiosity, could this rule go in the common file? Since it's the same across environments?

Reply:
It's *not quite* the same; prod has the aforementioned `extractSass.extract({` wrapper in it's `use` clause. Is it possible to define partial rules in common, then modify the `use` portion in the specialized files?

Reply:
I tried to get webpack-merge to do something similar to what you are suggesting in Insights, but I couldn't get it to work. I don't think it's smart enough to do that unfortunately.